### PR TITLE
Add conformer 1110

### PR DESCRIPTION
**Pull request by mmglove**
### PR types

Others
### PR changes

Others
### Describe

update benchmark case

### CODE REVIEW

File: tests/benchmark/conformer/run_benchmark.sh [hunk 1]
```diff
@@ -12,17 +12,24 @@ function _set_params(){
...
     fp_item=${8:-"fp32"}
-    TRAIN_LOG_DIR=${9:-$(pwd)}
-
+    model_item=${9:-"conformer"}
     benchmark_max_step=0
-
     run_log_path=${TRAIN_LOG_DIR:-$(pwd)}  # TRAIN_LOG_DIR 后续QA设置该参数
+# 添加日志解析需要的参数
+    base_batch_size=${batch_size}
+    mission_name="语音合成"
```
Comment (Jackwaterveg) [inline tests/benchmark/conformer/run_benchmark.sh]: 语音识别

File: tests/benchmark/conformer/run.sh [hunk 2]
```diff
@@ -1,8 +1,13 @@
 
 # 提供可稳定复现性能的脚本，默认在标准docker环境内py37执行： paddlepaddle/paddle:latest-gpu-cuda10.1-cudnn7  paddle=2.1.2  py=37
 # 执行目录：需说明
-CUR_DIR=${PWD}
-source ../../../tools/venv/bin/activate
+CUR_DIR=${PWD} # PaddleSpeech/tests/benchmark/conformer
+cd ../../../
+pip install -e .   # 安装pdspeech
```
Comment (Jackwaterveg) [inline tests/benchmark/conformer/run.sh]: pip Install -e 感觉应该放在prepare.sh里

File: tests/benchmark/conformer/run_benchmark.sh (hunk 1 above)
Comment (mmglove) [reply tests/benchmark/conformer/run_benchmark.sh]: done

File: tests/benchmark/conformer/run.sh (hunk 2 above)
Comment (mmglove) [reply tests/benchmark/conformer/run.sh]: done

**Review by Jackwaterveg — APPROVED** [review]
LGTM

**Pull request merged**